askama/Cargo.toml

```diff
@@ -36,8 +36,6 @@ with-warp = ["askama_derive/warp"]
 askama_derive = { version = "0.11.0", path = "../askama_derive" }
 askama_escape = { version = "0.10", path = "../askama_escape" }
 askama_shared = { version = "0.12.0", path = "../askama_shared", default-features = false }
-mime = { version = "0.3", optional = true }
```

Comment:
I think we should release this in a semver-compatible way, so this stuff should stay.